common/unified-chart/values.yaml

```diff
@@ -14,6 +14,24 @@ argoRollouts:
   activeMetadata:
     labels:
       role: active
+  progressiveDelivery:
+    enabled: false
+    stableMetadata:
+      labels:
+        role: stable
+    canaryMetadata:
+      labels:
+        role: canary
+    blackBox:
+      url: http://nexus.default.svc.cluster.local/blackbox
```

Comment:
will nexus be deployed in the default namespace?
shouldnt it be in frontegg namespace?